src/BrutForceForCaesarCipher.java

```diff
@@ -0,0 +1,148 @@
+            System.out.println("Check that the coding file in encoding UTF-8");
+            throw new RuntimeException(e);
+        }
+    }
+
+    //Manual hack
+    private static StringBuilder startManualBrutForce(Path path, int usingAlphabetSize){
+        try {
+            //Get data from file
+            List<String> dataFromFile = Files.readAllLines(path);
+            int counter = 0;
+            //Get line from data for validation
+            int lineNumberToRead = dataFromFile.size() > 1 ? 1 : 0;
+            while (counter <= usingAlphabetSize){
+                //Decode data
+                StringBuilder decodedLineFromFile = CaesarCipher.decoder(counter, dataFromFile.get(lineNumberToRead), alphabet);
+                System.out.printf("Iteration %d of %d%n".concat("Is this text correct?%n"), counter, usingAlphabetSize);
+                System.out.println("---> " + decodedLineFromFile);
+                //Manual validate result
+                String chosenOption = evaluateBrutForceResult();
+                if(chosenOption.equalsIgnoreCase("1")){
+                    break;
+                }
+                counter++;
+            }
+            //Decode all data
+            return CaesarCipher.decoder(counter, String.join("\n", dataFromFile), alphabet);
+        } catch (IOException e) {
+            System.out.println("Check that the coding file in encoding UTF-8");
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    //TODO Will be done in the future when ai analyzer will be done
+    private static StringBuilder startBrutForceWithAnalyzer(Path path, int usingAlphabetSize){
+        try {
+            String dataFromFile = Files.readString(path);
+            int counter = 0;
+            while (counter <= usingAlphabetSize){
+                StringBuilder decodedDataFromFile = CaesarCipher.decoder(counter, dataFromFile, alphabet);
+                int hackResult = HackAnalyzer.analyzeForComplianceWithTheMainPatterns(decodedDataFromFile);
+                if (hackResult == 0) {
+                    System.out.println("The result may be inaccurate :()\n"
+                            .concat("If decoded file is incorrect y can try again decode this decoded file")
+                    );
+                    return decodedDataFromFile;
+                } else if (hackResult > 0){
+                    System.out.println("Decodering complete");
+                    return decodedDataFromFile;
+                }
+                counter++;
+            }
+        }
+        catch (IOException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    private static String choiseOfBrutForceMethod(){
+        System.out.println("Choose the brut force item method to decode:\n"
+        .concat("1. Manual\n")
+        .concat("2. With auto analyzer"));
+        Scanner input = new Scanner(System.in);
+        String selectedItem = input.nextLine();
+        while (true){
```

Comment:
please do not use while-try, maybe add a boolean field

Reply:
Why? It's easy construction in this case..
is this better? 7dae7c6e
```suggestion
        String choice = INCORRECT_CHOICE;
        while (choice.equals(INCORRECT_CHOICE)){
            String selectedItem = input.nextLine();
            choice = switch (selectedItem){
                case "1" -> MANUAL_BRUT_FORCE;
                case "2" -> BRUT_FORCE_WITH_ANALYZER;
                default -> {
                    System.out.println(INCORRECT_CHOICE);
                    yield INCORRECT_CHOICE;
                }
            };
        }
        return choice;
```